tools/Google.Cloud.Tools.Analyzers/DefaultLiteralAnalyzer.cs

```diff
@@ -56,6 +56,19 @@ private static void AnalyzeDefaultLiteral(SyntaxNodeAnalysisContext context)
                 if (underlyingType.IsPrimitive() ||
                     underlyingType.BaseType?.SpecialType == SpecialType.System_Enum)
                 {
+                    // Check to see if https://github.com/dotnet/roslyn/issues/22578 has been fixed so we don't warn unnecessarily on parameters.
```

Comment:
Nice :)

(Alternatively, maybe we should add a warning at that point that the analyzer could be removed - although perhaps the tests will then start failing anyway, because it won't have the expected diagnostic?)

Reply:
Yeah, I think it's best to just rely on the tests failing to know whether to remove the analyzer.